assets/js/main.js

```diff
@@ -0,0 +1,308 @@
+      showConfirmButton: false,
+      timer: 1500,
+    });
+    addTaskButton.setAttribute("disabled", "disabled");
+  } else {
+    updateTask();
+    // sweet alert
+    Swal.fire({
+      position: "top-center",
+      icon: "success",
+      title: "The task has been updated successfully",
+      showConfirmButton: false,
+      timer: 1500,
+    });
+    document.getElementById("addTaskButton").innerHTML = "Add-Task";
+  }
+
+  displayData();
+  counter();
+  clear();
+};
+// add task function
+function addTask() {
+  var done = 0;
+  var taskObject = {
+    done: done,
+    name: taskName.value,
+    assignee: assignee.value,
+  };
+  tasksArray.push(taskObject);
+  localStorage.setItem("tasksList", JSON.stringify(tasksArray));
+}
+// display Data in web page
+function displayData() {
+  var result = "";
+  for (let i = 0; i < tasksArray.length; i++) {
+    if (tasksArray[i].done == 0) {
+      result += `
+          <div class="taskDiv d-flex justify-content-between">
+      
+          <div class="taskAndAssignee">
+          <p class="taskNamePara pb-2">${tasksArray[i].name}</p>
+          <p class="assigneePara">${tasksArray[i].assignee}</p>
+          </div>
+      
+          <div class="icons">
+          <a onclick="deleteTask(${i})" class="deleteIcon d-block text-danger fs-5" href="#"><i class="fa fa-trash" aria-hidden="true"></i></a>
+          <a onclick="getTaskData(${i})" class="readIcon d-block text-warning fs-5" href="#"><i class="fa fa-id-card" aria-hidden="true"></i></a>
+          <a onclick="TaskDone(${i})" class="readIcon d-block text-success fs-5" href="#"><i class="fa fa-check-circle" aria-hidden="true"></i></a>
+
+          </div>
+      
+          </div>
+      
+          `;
+    }
+  }
+  tasksData.innerHTML = result;
+}
+// display Data done in web page
+function displayDataConfirmed() {
+  var result = "";
+
+  for (let i = 0; i < tasksArray.length; i++) {
+    if (tasksArray[i].done == 1) {
+      result += `
+          <div class="taskDiv d-flex justify-content-between">
+      
+          <div class="taskAndAssignee">
+          <p class="taskNamePara pb-2">${tasksArray[i].name}</p>
+          <p class="assigneePara">${tasksArray[i].assignee}</p>
+          </div>
+      
+          <div class="icons">
+          <a onclick="deleteTask(${i})" class="deleteIcon d-block text-danger fs-5" href="#"><i class="fa fa-trash" aria-hidden="true"></i></a>
+          <a onclick="getTaskData(${i})" class="readIcon d-block text-warning fs-5" href="#"><i class="fa fa-id-card" aria-hidden="true"></i></a>
+          <a  class="readIcon d-block text-success fs-5" href="#"><i class="fa fa-check-circle" aria-hidden="true"></i></a>
+          
+          </div>
+      
+          </div>
+      
+          `;
+    }
+  }
+  tasksDataDone.innerHTML = result;
+}
+
+// clear data after add task
+function clear() {
+  taskName.value = " ";
+  assignee.value = " ";
+}
+
+// delete task when click on delete icon
```

Comment:
this comment here is not needed. The function name tells me already what is happening